test/functional/lua/fs_spec.lua

```diff
@@ -354,10 +364,6 @@ describe('vim.fs', function()
       command('edit test/functional/fixtures/tty-test.c')
     end)
 
-    after_each(function()
-      command('bwipe!')
-    end)
-
```

Comment:
?

Reply:
Something slipped in during a bad rebase. I'm fixing it.